valevas6/test
# Adding: Antibot, Honeypot, Recaptcha, SMTP and Config Split.

### Test Antibot 

1.  [Disable Javascript](https://www.computerhope.com/issues/ch000891.htm) in your browser.
2. Go to apply form in the project.
3. If javascript is disable the form will show you _You must have JavaScript enabled to use this form._

### Test HoneyBot
Honeybot prevents spam bots from completing the form. The module create an element in the form that only bots see and fill, also if the form is completed in less than 5 seconds it is considered as spam bots.

### Test ReCaptcha
1. In the job application form you will see a captcha option, you must put the letters you see in the image in the blank field.
2. If the captcha option is not completed you wont be able go to the next step.

### Test SMTP 
1. [Config you email and browser Gmail SMTP server.](https://www.hostinger.es/tutoriales/como-usar-el-servidor-smtp-gmail-gratuito/)
2. You can try sending an mail in home>>administration>>Configuration>>System>>SMTP Authentication Support, in the last field _Send test e-mail_ you put your email direction and the test email will be sent.

### Test Config Split
1. Add a new module, view, taxonomy or a new configuration.
2. Go to Configuration >> Development >> Configuration synchronization.
3. In core.extension you will see what news you have added since the last export.
4. _Import all_ will remove the changes you have made.
5. For example: Add a new module, go to conf synchronization in configuration and _import all_, verify the module and it will be uninstalled.
